### ISSUE TITLE
problem with selecting a stop in the Map tab with voiceover turned on

### ISSUE THREAD

**Issue by alanborning** [OneBusAway/onebusaway-iphone#1401]
bug report from one of our voiceover users:

I’ve noticed that when I try to select a stop in the Map tab with voiceover turned on, I cannot open the stops page to view upcoming bus arrival times. I would expect to be able to open the stops page by double tapping on the stop label. The map tab used to include a list view which extracted a list of stops from the map and listed them to make it easier for voiceover users to read and activate nearby stops. Perhaps this feature should be restored.
